src/Resources/laravel-debugbar-dark-mode.css

```diff
@@ -31,8 +31,11 @@ div.phpdebugbar code.phpdebugbar-widgets-sql span.hljs-keyword,
 div.phpdebugbar-openhandler .phpdebugbar-openhandler-header,
 div.phpdebugbar-openhandler .phpdebugbar-openhandler-header a {
     color: var(--color-gray-200);
+    text-shadow: 1px 1px #000;
```

Comment:
Maybe here??

Reply:
Yeah I added that to make the text in the timeline waterfall better readable on blue background:
<img width="699" alt="Screenshot 2024-04-03 at 20 22 30" src="https://github.com/barryvdh/laravel-debugbar/assets/18613261/77639078-bf56-4808-bc1c-a5b1b0ae6994">

With the text shadow:
<img width="861" alt="Screenshot 2024-04-03 at 20 28 27" src="https://github.com/barryvdh/laravel-debugbar/assets/18613261/662f5df4-c40b-460d-9c8c-42f1569402d9">

But quite some elements are affected by that change, perhaps too many:
<img width="861" alt="Screenshot 2024-04-03 at 20 28 27" src="https://github.com/barryvdh/laravel-debugbar/assets/18613261/d1aaddd9-3a62-4e74-a5e3-01f1f8cb4de3">

Reply:
>I added that to make the text in the timeline waterfall better readable

Since there are many classes involved(_also main `div.phpdebugbar`_), I saw it everywhere.